feature-configs/5g-ref-nw/profile-base/ran-general/kustomization.yaml

```diff
@@ -1,2 +1,3 @@
 resources:
+- ../../../deploy/container-mount-namespace
 - sctp 
```

Comment:
Why are we referring to the outer reference `../../../deploy/container-mount-namespace` but a local `sctp`?